std-bits/table/src/main/java/org/enso/table/operations/AddGroupNumber.java

```diff
@@ -230,11 +230,7 @@ private double sum() {
 
   private static final int STANDARD_DEVIATION_GROUP_COUNT = 5;
   private static final String[] STANDARD_DEVIATION_GROUP_LABELS = {
-    "Low (-2.5 to -1.5 sd)",
-    "Below Average (-1.5 to -0.5 sd)",
-    "Average (-0.5 to 0.5 sd)",
-    "Above Average (0.5 to 1.5 sd)",
-    "High (1.5 to 2.5 sd)"
+    "Very Low", "Low", "Average", "High", "Very High"
```

Comment:
Help needs updating for Table.add_group_number to include these ranges.

Reply:
and a typo while you are in that help

table2 = tabble.add_group_number (..Equal_Count 3) "g"